src/org/traccar/Context.java

```diff
@@ -137,10 +137,13 @@ public static EventForwarder getEventForwarder() {
     public static void init(String[] arguments) throws Exception {
 
         config = new Config();
-        if (arguments.length > 0) {
-            config.load(arguments[0]);
+        if (arguments.length <= 0) {
+            System.out.println("No configuration file given.");
+            System.exit(0);
```

Comment:
I think it's better to use RuntimeException, but yes, I like they idea of just throwing it here. Also, I think it would be better to use "Configuration file is not provided" as a text.